packages/synapse-react-client/src/components/AccessRequirementList/AccessRequirementList.tsx

```diff
@@ -257,50 +256,38 @@ export default function AccessRequirementList(
 
   const content = (
     <>
-      <DialogTitle>
-        <Stack direction="row" alignItems={'center'} gap={'5px'}>
-          {dialogTitle}
-          <Box sx={{ flexGrow: 1 }} />
-          <IconButton onClick={onHide}>
-            <IconSvg icon={'close'} wrap={false} sx={{ color: 'grey.700' }} />
-          </IconButton>
-        </Stack>
-      </DialogTitle>
-
-      <DialogContent>
```

Comment:
Remove the `DialogTitle`, `DialogContent` components from the `content` variable. It now just contains what was inside `DialogContent`